components/mintmaker/staging/base/kustomization.yaml

```diff
@@ -9,9 +9,9 @@ resources:
 namespace: mintmaker
 
 images:
-- name: quay.io/konflux-ci/mintmaker
-  newName: quay.io/konflux-ci/mintmaker
-  newTag: 7f0cd9e10e0ede8e2dcce8e5cbaa3647b4548cc4
+- name: quay.io/repository/redhat-user-workloads/konflux-mintmaker-tenant/mintmaker-refactoring-pipelinerun
```

Comment:
`quay.io/konflux-ci/mintmaker` is our controller image.
You can't change this, the `name` is the image kustomize should transform for you, if you use this new image, kustomize won't find anything to be transformed.